omod/src/main/webapp/resources/htmlforms/HMIS-HIV-003-HivCareArtCard-ClinicalAssessmentPage.xml

```diff
@@ -5,577 +5,727 @@
-                        disable_fields("other-reason-for-next-appointment");
-                    }
-                });
+        var instabilityCause = "";
 
-                disable_fields("edd");
-                disable_fields("emtct");
-                disable_fields("age-in-months");
-                disable_fields("anc_no");
-
-                /* enable the edd,emtct,age in months and ANC no. fields in case the client is pregnant */
-
-                jq(".pregnancy").change(function () {
-                    if (jq(this).find('#pregnant').find(':checkbox').prop('checked')) {
-                        enable_fields("edd");
-                        enable_fields("emtct");
-                        enable_fields("age-in-months");
-                        enable_fields("anc_no");
-                    } else {
-                        disable_fields("edd");
-                        disable_fields("emtct");
-                        disable_fields("age-in-months");
-                        disable_fields("anc_no");
-                    }
-                });
+        jq(document).ready(function () {
 
-                jq(".pregnancy").change(function () {
-                    if (jq(this).find('#not-pregnant-not-breast-feeding').find(':checkbox').prop('checked')) {
-                        jq("#pregnant").find("input[type$='checkbox']").prop("checked", false);
-                        jq("#breast-feeding").find("input[type$='checkbox']").prop("checked", false);
-                        disable_fields("edd");
-                        disable_fields("emtct");
-                        disable_fields("age-in-months");
-                        disable_fields("anc_no");
-                    }
-                });
+        jq("#visit-date").addClass("html-form-entry-disabled-field");
+        <!-- Determine muac code based on muac score and age -->
+        var age  =  <lookup complexExpression="$patient.age"/>;
+        getMUACCodeFromMUACScoreByAge(age, '1343', '99030');
 
-                jq(".pregnancy").change(function () {
-                    if (jq(this).find('#not-pregnant-not-applicable').find(':checkbox').prop('checked')) {
-                        jq("#pregnant").find("input[type$='checkbox']").prop("checked", false);
-                        jq("#breast-feeding").find("input[type$='checkbox']").prop("checked", false);
-                        jq("#not-pregnant-not-breast-feeding").find("input[type$='checkbox']").prop("checked", false);
-                        disable_fields("edd");
-                        disable_fields("emtct");
-                        disable_fields("age-in-months");
-                        disable_fields("anc_no");
-                    }
-                });
+        //Block Multiple encounter entry on same date
+        blockEncounterOnSameDateEncounter(jq("#encounterDate").find("input[type=hidden]"),"block");
 
-                /*enable disable family planning options*/
+        var stableArray = [];
```

Comment:
@slubwama what is this for?

Reply:
an array of checkes that makeup the final check if a patient is stable or not. (DSDM)

Reply:
done